### PR TITLE
Search Beta 

### PR DESCRIPTION
**Pull request by xdavidson**
repo based search

### CODE REVIEW

File: pantheon-bundle/frontend/src/app/searchBeta.tsx [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+            />
+            <Button variant={ButtonVariant.control} aria-label="search button for search input">
+              <SearchIcon />
+            </Button>
+          </InputGroup>
+        </ToolbarItem>
+        <ToolbarGroup variant="filter-group">
+          <ToolbarFilter
+            chips={filters.status}
+            // deleteChip={this.onDelete}
```
Comment (grantsydney) [inline pantheon-bundle/frontend/src/app/searchBeta.tsx:232]: do you need the `onDelete` function here?

File: pantheon-bundle/frontend/src/app/searchBeta.tsx [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+              onToggle={this.onStatusToggle}
+              onSelect={this.onStatusSelect}
+              selections={filters.status}
+              isOpen={statusIsExpanded}
+              placeholderText="Status"
+            >
+              {statusMenuItems}
+            </Select>
+          </ToolbarFilter>
+          {/* <ToolbarFilter chips={filters.ctype} deleteChip={this.onDelete} categoryName="Content Type"> */}
```
Comment (grantsydney) [inline pantheon-bundle/frontend/src/app/searchBeta.tsx:248]: Is this duplicate code? Looks the same starting on line 249

File: pantheon-bundle/frontend/src/app/searchBeta.tsx (hunk 1 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/searchBeta.tsx:232]: Currently, users have two options to remove filter chips

-  "clear all filters" 

-  uncheck items from the dropdown

ideally, we enable deleteChip prop for ToobarFilter component. This gives user a third option to remove chip. I created a jira ticket to capture the effort: CCS-3975

File: pantheon-bundle/frontend/src/app/searchBeta.tsx (hunk 2 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/searchBeta.tsx:248]: This is not duplicated code, line 248 enables "deleteChip" function while line 249 doesn't. Ideally, we want to have "deleteChip" enabled. this work effort is captured in CCS-3975

Comment (grantsydney) [reply pantheon-bundle/frontend/src/app/searchBeta.tsx:248]: Ah got it, thanks for clarifying

**Review by grantsydney — DISMISSED** [review]

**Review by grantsydney — APPROVED** [review]

**Pull request merged**